### PR TITLE
Added Travis config

### PR DESCRIPTION
**Pull request by ViniTou**
~DO NOT MERGE BEFORE REMOVING TMP COMMIT.~

Can be merged now :)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    - php: 7.2
+      env: TEST_CONFIG="phpunit.xml"
+    - php: 7.2
+      env: REST_TEST_CONFIG="phpunit-integration-rest.xml" SYMFONY_ENV=behat SYMFONY_DEBUG=1 SF_CMD="ez:behat:create-language 'pol-PL' 'Polish (polski)'"
+# 7.3
+    - php: 7.3
+      env: TEST_CONFIG="phpunit.xml"
+    - php: 7.3
+      env: REST_TEST_CONFIG="phpunit-integration-rest.xml" SYMFONY_ENV=behat SYMFONY_DEBUG=1 SF_CMD="ez:behat:create-language 'pol-PL' 'Polish (polski)'"
+    - name: "Code Style Check"
```
Comment (alongosz) [inline .travis.yml]: Please set explicitly php 7.3 for this job for performance. Job itself shows something that looks like warning
> This job ran using the default PHP version, which was updated to 7.2 on March 12th, 2019

so I'd say it's not best practice to have implicit version.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+env:
+  global:
+    # For functional and acceptance tests
+    - COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/selenium.yml"
+    # In order to specify extra flags like ignoring platform reqs, using only stable packages and so on.
+    - COMPOSER_FLAGS=""
```
Comment (mnocon) [inline .travis.yml]: Can be removed?

Comment (ViniTou) [reply .travis.yml]: I would leave it here just for sake of consistency and future options

**Comment by ViniTou** [comment]
Yup, I tried to setup metarepo for different PRs and now we are back to square one with it.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  - TEST_TIMEZONES=("America/New_York" "Asia/Calcutta" "UTC")
+  - TEST_TIMEZONE=${TEST_TIMEZONES["`shuf -i 0-2 -n 1`"]}
+
+install:
+  - if [ "$CHECK_CS" == "1" -o "$TEST_CONFIG" != "" ]; then travis_retry composer install --prefer-dist --no-interaction --no-suggest ; fi
+
+# execute phpunit or behat as the script command
+script:
+  - if [ "$TEST_CONFIG" != "" ] ; then php -d date.timezone=$TEST_TIMEZONE -d memory_limit=-1 vendor/bin/phpunit -c $TEST_CONFIG ; fi
+  - if [ "$REST_TEST_CONFIG" != "" ] ; then cd "$HOME/build/ezplatform"; docker-compose exec --user www-data app sh -c "php -d date.timezone=$TEST_TIMEZONE -d memory_limit=-1 bin/phpunit --bootstrap vendor/autoload.php -c vendor/ezsystems/ezplatform-rest/$REST_TEST_CONFIG"  ; fi
```
Comment (alongosz) [inline .travis.yml]: Follow up for later: install dependencies for ezplatform-rest and run tests from there.

**Comment by alongosz** [comment]
@ViniTou let's rebase this so I can merge it. I don't think we'll solve some of those issues until everything is ready.

**Comment by ViniTou** [comment]
@alongosz 
done!

**Pull request merged**